openstack/smn/v2/logtank/urls.go

```diff
@@ -0,0 +1,21 @@
+package logtank
+
+import (
+	"github.com/chnsz/golangsdk"
+)
+
+func createURL(c *golangsdk.ServiceClient, topicUrn string) string {
+	return c.ServiceURL("topics", topicUrn, "logtanks")
+}
+
+func getURL(c *golangsdk.ServiceClient, topicUrn string) string {
+	return c.ServiceURL("topics", topicUrn, "logtanks")
+}
+
+func updateURL(c *golangsdk.ServiceClient, topicUrn string, logTankId string) string {
+	return c.ServiceURL("topics", topicUrn, "logtanks", logTankId)
+}
+
+func DeleteURL(c *golangsdk.ServiceClient, topicUrn string, logTankId string) string {
+	return c.ServiceURL("topics", topicUrn, "logtanks", logTankId)
+}
```

Comment:
this too, you can call it resourceURL,
which means the URL contains resource ID

Reply:
renamed resourceURL